lib/frontier/controller_action/index_action.rb

```diff
@@ -6,14 +6,37 @@ def to_s
     raw = <<-STRING
 def index
   #{Frontier::Authorization::Assertion.new(model_configuration, :index).to_s}
-  #{model_configuration.as_ivar_collection} = #{scopable_object}.page(params[:page])
+#{Frontier::RubyRenderer.new(object_scope).render(1)}
```

Comment:
See [this blog post](http://blog.jayfields.com/2006/12/ruby-multiline-strings-here-doc-or.html) for more info about heredoc, which is what line 6 is initiating and line 11 is ending.